sched/plan_class_spec.h

```diff
@@ -44,6 +44,8 @@ struct PLAN_CLASS_SPEC {
     regex_t os_regex;
     bool have_cpu_vendor_regex;
     regex_t cpu_vendor_regex;
+    bool have_cpu_model_regex;
+    regex_t cpu_model_regex;
```

Comment:
Neither variable is initialised or read from XML.

Reply:
Oops!  Fixed - thanks.

Reply:
`have_cpu_model_regex` is still uninitialised.